src/three-render-objects.js

```diff
@@ -183,10 +185,12 @@ export default Kapsule({
     tbControls: state => state.controls // to be deprecated
   },
 
-  stateInit: () => ({
+  stateInit: ({ cameraType = 'perspective' }) => ({
     scene: new three.Scene(),
-    camera: new three.PerspectiveCamera(),
-    clock: new three.Clock()
+    clock: new three.Clock(),
+    camera: cameraType === 'perspective' ?
```

Comment:
With this logic if the user specifies anything else other than `perspective` it will fall back to orthographic, which is a little dangerous. Let's swap that so that `perspective is the default fallback`:
```js
camera: new three[cameraType === 'orthographic' ? 'OrthographicCamera' : 'PerspectiveCamera']();
```